managed/ui/src/components/config/Storage/StorageConfigurationContainer.js

```diff
@@ -31,6 +32,21 @@ const mapDispatchToProps = (dispatch) => {
   return {
     addCustomerConfig: (config) => {
       return dispatch(addCustomerConfig(config)).then((response) => {
+        if (response.error) {
+          const errorMessageObject = response.payload?.response?.data?.error || response.payload.message;
+          Object.keys(errorMessageObject).forEach((errorKey) => {
+            toast.error(
+              <ol>
+                {errorMessageObject[errorKey].map((error) => (
+                  <li>{error}</li>
+                ))}
+              </ol>
+            );
+          });
+        }
+        else {
+        toast.success("Successfully added the backup configuration.");
```

Comment:
We can have this message also from BE.

Reply:
As discussed overs call we don't have it from BE